wagtail/documents/forms.py

```diff
@@ -61,6 +61,13 @@ def save(self, commit=True):
     class Meta:
         widgets = {"tags": AdminTagWidget, "file": forms.FileInput()}
 
+    def clean_tags(self):
+        tags = self.cleaned_data["tags"]
+        max_tag_length = 100
```

Comment:
Maybe it would be better to get `max_tag_length` from Tag or TagBase as it is suggested in the issue comments:
```
max_length = TagBase._meta.get_field("name").max_length
if any(len(tag) > max_length for tag in tags):
```

Reply:
@halitcelik thanks for the suggestion!